src/include/network/remote/dialogs.rb

```diff
@@ -60,9 +60,17 @@ def RemoteMainDialog
           # RadioButton label
           Left(
             RadioButton(
-              Id(:allow),
-              _("&Allow Remote Administration"),
-              Remote.IsEnabled
+              Id(:allow_with_vncmanager),
+              _("&Allow Remote Administration With Session Management"),
```

Comment:
We should write some explanation of the tradeoffs between the two in the help text. As an ignorant user, why would I ever want the worse option (without secession malignment or whatsit called)?